src/pages/EditMonitor.vue

```diff
@@ -300,6 +302,12 @@ export default {
                 this.$root.getSocket().emit("getMonitor", this.$route.params.id, (res) => {
                     if (res.ok) {
                         this.monitor = res.monitor;
+
+                        for (let i = 0; i < this.$root.notificationList.length; i++) {
+                            if (this.$root.notificationList[i].isDefault == true) {
+                                this.monitor.notificationIDList[this.$root.notificationList[i].id] = true;
+                            }
+                        }
```

Comment:
Haven't pull this to try yet, but should we put this in `isAdd?` section instead? Otherwise if the user disabled a default notification, next time he edits the monitor it will be enabled again?

Reply:
Yes thank you! I forgot `isAdd` exists.